front/app/modules/commercial/native_surveys/admin/components/SurveyBuilderSettings/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+// styles
+import styled from 'styled-components';
+import { colors } from 'utils/styleUtils';
+
+// components
+import { Title, Box, stylingConsts } from '@citizenlab/cl2-component-library';
+import Button from 'components/UI/Button';
+import CloseIconButton from 'components/UI/CloseIconButton';
+
+// intl
+import messages from './messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+const StyledBox = styled(Box)`
+  box-shadow: -2px 0px 1px 0px rgba(0, 0, 0, 0.06);
+`;
+
+const StyledCloseIconButton = styled(CloseIconButton)`
+  position: absolute;
+  right: 8px;
+`;
```

Comment:
Would we be able to assign these on a Box that wraps the CloseIconButton instead? Iva has mentioned to me that we should try not to use styled components when possible (not sure if it's possible in this case though?)

Reply:
Did she mention why? It might be something we can solve another way

Reply:
I think it's because it can make the code cluttered/longer. It's not really an issue here maybe, but in some other components we have a ton of styled components first, and it makes it really confusing when updating things afterwards.

Reply:
Oh okay In that case, an alternative solution if it happens a lot would be allow more of those props on those components. I'll change this one here though. We can tackle / plan for other such cases separately